types/react-widgets/index.d.ts

```diff
@@ -14,10 +14,10 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-export import Calendar = require('./lib/Calendar');
-export import Combobox = require('./lib/Combobox');
-export import DateTimePicker = require('./lib/DateTimePicker');
-export import DropdownList = require('./lib/DropdownList');
-export import Multiselect = require('./lib/Multiselect');
-export import NumberPicker = require('./lib/NumberPicker');
-export import SelectList = require('./lib/SelectList');
+export import Calendar = require("./lib/Calendar");
```

Comment:
What the point to commit these changes for codestyle? :)

Reply:
The README suggested I run Prettier on the package folder, so I committed all the changes. https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes

Reply:
Gotcha! Thanks for your answer :)